concepticondata/conceptlists/Lai-2023-291.tsv

```diff
@@ -0,0 +1,292 @@
+Lai-2023-291-1	41	the dust	2	DUST	45	the dust
+Lai-2023-291-2	55	firewood	10	FIREWOOD	534	firewood
+Lai-2023-291-3	145	noon	12	MIDDAY	750	noon // the noon
+Lai-2023-291-4	149	the otter	15	OTTER	317	the otter
+Lai-2023-291-5	208	that	78	THAT	947	that
+Lai-2023-291-6	2	all	98	ALL	962	all
+Lai-2023-291-7	265	here	136	HERE		here
+Lai-2023-291-8	372	to burn [transitive]	141	BURN (SOMETHING)		to burn [transitive]
+Lai-2023-291-9	92	the hoof	152	HOOF	264	the hoof
+Lai-2023-291-10	158	red	156	RED	1007	red
+Lai-2023-291-11	14	black	163	BLACK	1005	black
+Lai-2023-291-12	16	to blow (of wind)	175	BLOW (OF WIND)	1738	to blow (of wind)
+Lai-2023-291-13	17	the body hair (hair or fur)	189	HAIR (BODY)	266	the body hair (hair or fur) // the hair
+Lai-2023-291-14	278	other	197	OTHER		other
+Lai-2023-291-15	116	to lie down	215	LIE DOWN	1661	to lie down
+Lai-2023-291-16	54	the fire	221	FIRE	18	the fire
+Lai-2023-291-17	56	the fish	227	FISH	351	the fish
+Lai-2023-291-18	58	the flea	232	FLEA	355	the flea
+Lai-2023-291-19	61	the flower	239	FLOWER	377	the flower
+Lai-2023-291-20	114	left	244	LEFT	710	left
+Lai-2023-291-21	63	the fog	249	FOG	16	the fog
+Lai-2023-291-22	144	the nit	267	NIT	357	the nit
+Lai-2023-291-23	105	to knead	274	KNEAD	1594	to knead
+Lai-2023-291-24	115	to lick	319	LICK	1674	to lick
+Lai-2023-291-25	23	to chew	321	CHEW	1424	to chew
+Lai-2023-291-26	182	the sickle	341	SICKLE	624	the sickle
+Lai-2023-291-27	65	the forest	420	FOREST	50	the forest
+Lai-2023-291-28	110	late	477	LATE	1019	late
+Lai-2023-291-29	118	the lip (the lips)	478	LIP	85	the lip (the lips)
+Lai-2023-291-30	385	to grow	479	GROW		to grow
+Lai-2023-291-31	181	shy	487	SHY	1365	shy
+Lai-2023-291-32	216	three	492	THREE	799	three
+Lai-2023-291-33	57	five	493	FIVE	801	five
+Lai-2023-291-34	69	the frog	503	FROG	349	the frog
+Lai-2023-291-35	113	to learn	504	LEARN	1742	to learn
+Lai-2023-291-36	241	the wolf	522	WOLF	324	the wolf
+Lai-2023-291-37	3	the ant	587	ANT	365	the ant
+Lai-2023-291-38	387	to hide (conceal)	602	HIDE (CONCEAL)		to hide (conceal)
+Lai-2023-291-39	77	the grass	606	GRASS	436	the grass
+Lai-2023-291-40	79	the hail	609	HAIL	13	the hail
+Lai-2023-291-41	95	the horse	615	HORSE	268	the horse
+Lai-2023-291-42	102	the ice	617	ICE	17	the ice
+Lai-2023-291-43	109	the lake	624	LAKE	31	the lake
+Lai-2023-291-44	112	the leaf	628	LEAF	376	the leaf
+Lai-2023-291-45	126	the meat	634	MEAT	443	the meat
+Lai-2023-291-46	132	the mountain	639	MOUNTAIN	23	the mountain
+Lai-2023-291-47	135	the mud	640	MUD	46	the mud
+Lai-2023-291-48	295	the ashes	646	ASH		the ashes
+Lai-2023-291-49	156	the rain	658	RAIN (PRECIPITATION)	10	the rain
+Lai-2023-291-50	162	the river	666	RIVER	30	the river
+Lai-2023-291-51	163	the road	667	ROAD	38	the road
+Lai-2023-291-52	164	the root	670	ROOT	375	the root
+Lai-2023-291-53	170	the sand	671	SAND	44	the sand
+Lai-2023-291-54	43	early	672	EARLY	1018	early
+Lai-2023-291-55	134	the mouth	674	MOUTH	84	the mouth
+Lai-2023-291-56	250	the shit	676	SHIT (DEFECATE)	151	the shit
```

Comment:
The current mapping refers to the action. I believe [FAECES (EXCREMENT)](https://concepticon.clld.org/parameters/675) would be a more suitable mapping for this gloss.

Reply:
Yes, this is a wrong mapping.